src/SDK/SmartDetectorsShared/Trace/IExtendedTracer.cs

```diff
@@ -15,18 +15,6 @@ namespace Microsoft.Azure.Monitoring.SmartDetectors.Trace
     /// </summary>
     public interface IExtendedTracer : ITracer
```

Comment:
>IExtendedTracer [](start = 21, length = 15)

how about we get rid of this altogether?

Reply:
Done

---
In reply to: [275233179](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/59#discussion_r275233179) [](ancestors = 275233179)